source/server/module-automation.js

```diff
@@ -0,0 +1,27 @@
+Space.Module.mixin({
+
+  Routers: [],
+  Projections: [],
+
+  onDependenciesReady: function() {
+    this._wrapLifecycleHook('onInitialize', this._onInitializeEventSourcing);
```

Comment:
hmm … let's keep it private for now, less API is always better :wink: you only need it for very specialized cases and if we find that we start using it like a public api we can still remove the underscore and switch to public later on